.cursor/rules/typescript.mdc

```diff
@@ -0,0 +1,37 @@
+---
+description:
+globs: *.ts, *.tsx
+alwaysApply: false
+---
+# TypeScript Guide
+
+- TypeScript version: 5.7.3
```

Comment:
nit: using a specific patch version might get outdated quite soon. Maybe we just state it as: TypeScript version: 5?